### ISSUE TITLE
Ensure template can be used with  cookiecutter cli 

### ISSUE THREAD

**Issue by MichaelClifford** [aicoe-aiops/project-template#41]

**Comment by MichaelClifford** [comment]
cc @goern

**Comment by goern** [comment]
It would be cool, if we can use the same cookie-cutter template within an argo workflow, so that we can use it as an initialization step of kebechet ...

**Comment by MichaelClifford** [comment]
@chauhankaranraj any update on this issue?

**Comment by chauhankaranraj** [comment]
> @chauhankaranraj any update on this issue?

Yep, the work-in-progress cookiecutter repo can be found [here](https://github.com/aicoe-aiops/cookiecutter-data-science). This can be used with the [cookiecutter](https://cookiecutter.readthedocs.io/en/latest/README.html) cli like this:
```bash
cookiecutter gh:aicoe-aiops/cookiecutter-data-science
```
